clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -929,6 +929,26 @@
     public static final Schema[] DELETE_TOPICS_REQUEST = new Schema[] {DELETE_TOPICS_REQUEST_V0};
     public static final Schema[] DELETE_TOPICS_RESPONSE = new Schema[] {DELETE_TOPICS_RESPONSE_V0};
 
+    /* Transactions API */
+    public static final Schema INIT_PRODUCER_ID_REQUEST_V0 = new Schema(
+            new Field("appid",
+                    NULLABLE_STRING,
+                    "The appid whose pid we want to retrieve or generate.")
+    );
+
+    public static final Schema INIT_PRODUCER_ID_RESPONSE_V0 = new Schema(
```

Comment:
Seems like we might need an error_code also. Then we can provide a trivial implementation of `InitPidRequest.getErrorResponse`.